report.qmd

```diff
@@ -100,110 +137,328 @@ Our derived subquestions, respective to each goal are then the following:
 
 ```
 
-**Scale-Specific Indicators for Stream Assessment**
+### Scale-Specific Indicators for Stream Assessment
 
 Urban stream analysis requires scale-appropriate methodologies. Different ecosystem services or hydrological characteristics demand different spatial extents. [@RANTA2021106980] uses for example 100m stretches for biological assessments and 500m buffers for hydromorphilogical evaluations.
 
 Therefore, when deciding on our scale, we considered our indicators and attempted to closely match them to retain a good representation of the spatial dynamics of the stream functions we aimed to analyze.
 
-**Impact of Scale on Detectable Effects**
+### Impact of Scale on Detectable Effects
 
 The spatial resolution of the data we collected significantly shapes the analytical results. Averaging the data collected within the chosen 100 x 100m cells allows for generalized insights, which we found best fitting. It may however still obscure localized effects such as even smaller micro climatic benefits of tree clusters or linear vegetation elements. We chose the trade off that best fits our and balances detail with analytical feasibility.
 
 // For the MCDA -\> add how we average and normalise values here?
 
-### Climate Adaptation
-
-### Biodiversity
+### Normalization
 
-For air quality, relabeling the RGB values to legend.
+The data used in this study comes from many different sources and uses different units as discussed above. Therefore, to compare them evenly, all values need to be brought to the same scale, and normalized in order to later conduct K-means clustering to discover appropriate typologies. We therefore could not use one method for all data normalization. Instead, each topic or data map uses its own way of normalizing respective to the data acquisition and availability. For climate adaptation, values like temperature and air pollution were compared to limits from the EU or WHO. For biodiversity, we used scales that show how suitable or rich an area is for nature. For quality of life, we used standards such as walking distances or access to green areas. Each method is explained in the following sections under their respective topic or map that they relate to.
 
-| Green Band Value | SO₂ Range (µg/m³) | Color        | Class Label | Class Value |
-|------------------|-------------------|--------------|-------------|-------------|
-| 168              | 0.0 – 0.1         | Dark Green   | Very Good   | 0           |
-| 201              | 0.1 – 0.2         | Light Green  | Good        | 1           |
-| 237              | 0.2 – 0.5         | Yellow       | Moderate    | 2           |
-| 204              | 0.5 – 1.0         | Light Orange | Poor        | 3           |
-| 102              | 1.0 – 2.0         | Dark Orange  | Very Poor   | 4           |
-| 0                | 2.0 – 5.0         | Red          | Hazardous   | 5           |
+## Results
 
 ### MCDA
 
-#### Maps
+### Maps
```

Comment:
I don't think you nead a heading "Maps". You would need, however, some introductory paragraph before you start the thre main sub-sections.